receiver/flinkmetricsreceiver/README.md

```diff
@@ -0,0 +1,39 @@
+# FlinkMetrics Receiver
+
+This receiver uses Flink's [REST API](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#rest-api-integration) to collect Jobmanager, Taskmanager, Job, Task and Operator metrics.
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
```

Comment:
If this is a brand new receiver, [Alpha](https://github.com/open-telemetry/opentelemetry-collector#alpha) might be the most appropriate stability level

Reply:
switched to Alpha